warehouse/templates/manage/organizations.html

```diff
@@ -72,14 +72,15 @@ <h3 class="organization-snippet__title">
             {% if organization.is_approved and organization.is_active %}
             <a href="{{ request.route_path('organizations.profile', organization=organization.name) }}">{{ organization.name }}</a>
             {% else %}{{ organization.name }}{% endif %}
-            {% if organization.orgtype == OrganizationType.Company and not organization.active_subscription %}
+
+            {% if organization.is_approved == None %}
```

Comment:
Should this be:
```suggestion
            {% if organization.is_approved is None %}
```
I recognize this was there already, but opportunity to fix